packages/site/src/types/content.ts

```diff
@@ -1,31 +1,98 @@
+// Define supported platform types (for iframes/rendering)
+export type PlatformType = "web" | "mobile";
+
+// Define supported component types (including versions)
+export type ComponentType = "web" | "webSupported" | "mobile";
```

Comment:
not sure I like this

"web" and "mobile" appearing in both types, and the fact that it's just "web" rather than "webLegacy" here

could be some room for improvement